tests/cases/fourslash/commentSelection1.ts

```diff
@@ -0,0 +1,26 @@
+// Simple comment selection cases.
+
+//// let var1[| = 1;
+//// let var2 = 2;
+//// let var3 |]= 3;
+//// 
+//// let var4[| = 4;|]
+//// 
+//// let [||]var5 = 5;
+//// 
+//// //let var6[| = 6;
+//// //let var7 = 7;
+//// //let var8 |]= 8;
+
+verify.commentSelection(
+    `//let var1 = 1;
+//let var2 = 2;
+//let var3 = 3;
+
+let var4/* = 4;*/

```

Comment:
We're following commenting implementation of VS. If we feel strongly about it I wouldn't mind changing it back, it would make the code a little bit cleaner.

Reply:
No, it should keep doing whatever it was doing.